ui/src/lib/labeller.ts

```diff
@@ -251,8 +251,11 @@ class StepLabeller implements StepMatcher<string> {
     return `Convert "${step.column}" into text`;
   }
 
-  hierarchy(_step: Readonly<S.HierarchyStep>) {
-    return `Aggregate geography`;
+  hierarchy(step: Readonly<S.HierarchyStep>) {
+    const prefix = `Set up a geographical hierarchy in "${step.hierarchyLevelColumn}"`;
+    const hierarchyString =
+      step.hierarchy.length > 1 ? ': ' + step.hierarchy.reduce((a, b) => `${a} > ${b}`) : '';
```

Comment:
I reckon using a `join` with `>` token is way easier to read (and common) than a `reduce`